solutions/secure/variables.tf

```diff
@@ -14,9 +14,16 @@ variable "resource_group_name" {
   description = "The name of a new or an existing resource group in which to provision the Databases for Elasicsearch in."
```

Comment:
This description needs to be updated to include details of what happens when prefix is added